niv-rozanowich/tvmaze/style.css

```diff
@@ -0,0 +1,1500 @@
+@font-face {
```

Comment:
is that the same `style.css` of the same icons of your root folder?
if so, use both of them from the same source instead of copy-pasting.